File: dart/condicional/2.dart
import 'dart:io';

void main() {
  print("Mayor o menor de edad");
  
  print("¿Cuál es su nombre?");
  
  var nombre = stdin.readLineSync();

  print("Digite su edad: ");

  int edad = int.parse(stdin.readLineSync()!);

  if(edad >= 18 && edad < 100){
      
      print("${nombre} Usted es mayor de edad.");
      
  }else if(edad <=0 || edad >=100){
      
      print("Digite una edad valida");
      
  }else{
      
      print("${nombre} Usted es menor de edad.");
      
  }
}

File: CondicionalesCsharp/2.cs
using System;


namespace Condicionales
{
    internal class Dos
    {
        public Dos() {
            Console.WriteLine("Ingrese su nombre: ");
            string nombre = Console.ReadLine();

            Console.WriteLine("Ingrese su edad");
            int edad = int.Parse(Console.ReadLine());

            if (edad < 1 || edad > 100)
            {
                Console.WriteLine("Señor(a) " + nombre + " ingrese una edad valida");
            }else if (edad >= 18 && edad <= 100) {
                Console.WriteLine("Señor(a) " + nombre + " usted es mayor de edad");
            }
            else
            {
                Console.WriteLine("Señor(a) " + nombre + "usted es menor de edad");
            }

        }
    }
}
